src/OpenLoco/src/Widget.cpp

```diff
@@ -273,11 +273,9 @@ namespace OpenLoco::Ui
             return;
         }
 
-        // TODO: this is odd most likely this is another flag like Widget::kImageIdColourSet
-        if (imageId.hasSecondary())
+        if (!isColourSet && imageId.hasSecondary())
         {
-            // 0x4CAE5F
-            assert(false);
+            imageId = imageId.withSecondary(colour.c());
         }
```

Comment:
NB: this was changed to accommodate the remap in the new brush image:
![00002367](https://github.com/OpenLoco/OpenLoco/assets/604665/8d45e567-4d9a-4ffa-8d31-a52ed49527cb)

See also this bit of code from the land tool's `prepareDraw` function:
```
auto landObj = ObjectManager::get<LandObject>(_lastSelectedLand);
auto pixelColour = static_cast<Colour>(Gfx::getG1Element(landObj->mapPixelImage)->offset[0]);
self.widgets[widx::paint_mode].image = Gfx::recolour2(ImageIds::paintbrush, Colour::white, pixelColour);
```